scripts/setup.sh

```diff
@@ -74,20 +75,20 @@ fi
 apt-get update -q
 apt-get install -t ${DEB_RELEASE}-backports -y vmdebootstrap qemu-utils
 
-# Copy know working vmdebootstrap version 0.10 from git
+# Copy known working vmdebootstrap version 0.10 from git
 cd $HOME
 git clone git://git.liw.fi/vmdebootstrap
 cd vmdebootstrap
 git checkout tags/vmdebootstrap-0.10
 cp vmdebootstrap /usr/sbin/vmdebootstrap
 
-cd $HOME
-git clone https://github.com/TheTorProject/lepidopter.git
-
 # Add loop kernel module required to mount loop devices
 modprobe loop
 
-cd lepidopter/
+cd $HOME/lepidopter/
+curl -L ${LEPIDOPTER_UPDATE_GIT}/tarball/master | tar zxvf - \
+    -C ${LEPIDOPTER_UPDATE_PATH} --wildcards */updater/ --strip-components=2 \
+    --exclude='latest_version' --exclude='test_updater.py' --exclude='example.py'
```

Comment:
Hum, I see what you are doing here actually. You want to include in the image also all the older update scripts. If that is the case then I guess this could work.